unstructured/partition/msg.py

```diff
@@ -67,3 +67,32 @@ def build_msg_metadata(msg_obj: msg_parser.MsOxMessage) -> ElementMetadata:
         subject=getattr(msg_obj, "subject", None),
         date=email_date,
     )
+
+
+def extract_attachment_info(
+    filename: str,
```

Comment:
Can we update this to accept `file` as well? And then you can use the `exactly_one` helper function instead of raising the `ValueError` in this function.